pages/ksphere/d2kf/1.0.1-0.3.0/introducing/index.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: layout.pug
+navigationTitle: Introducing D2KF
+title: Introducing D2KF
+excerpt: Deploy and manage machine learning models with ease
+menuWeight: 1
+---
+D2KF is a cloud-native suite of best-of-breed open-source technologies that allow data scientists to extract value from data immediately by providing a familiar environment for development and all the technologies needed to deploy and scale models in production, while keeping your valuable data safe. D2KF solves a key problem that enterprises face: How to get a return on your expensive AI investments? Going from a prototype to a deployment is often hard, but it does not have to be thanks to D2KF.
+
+D2iQ's D2KF leverages our expertise in Kubernetes, so companies can run their machine learning workloads anywhere: in the cloud, on premise, locally, or in hybrid environments.
```

Comment:
```suggestion
D2iQ's D2KF leverages our Expertise Kubernetes, Konvoy, so companies can run their machine learning workloads anywhere: in the cloud, on-premise, or in hybrid environments.
```

Reply:
Nits, I would remove `locally` because running via docker or KIND is more for testing